.env

```diff
@@ -0,0 +1,5 @@
+DB_URL=jdbc:mysql://db:3306/bms_db
```

Comment:
This file should not be pushed! make sure to remove it from this PR.

Reply:
.env fle is removed